yaml/com/ibm/Panel.interface.yaml

```diff
@@ -0,0 +1,63 @@
+description: >
+    Implements control panel activities.
+methods:
+    - name: TriggerPanelLampTest
+      description: >
+          API to trigger lamp test on panel.
+      parameters:
+          - name: status
+            type: boolean
+            description: >
+                Panel lamp test request.
+
+    - name: toggleFunctionState
+      description: >
+          Toggles the state of given panel functions.
```

Comment:
I would avoid this function as this function is used by PLDM to indicate to the panel that a function has changed in enablement state